docs/parse_argparse.py

```diff
@@ -310,21 +317,20 @@ def main():
         os.path.dirname(__file__), "source", "scripts")
     remove_existing_docs(rst_scripts_dirpath)
     remove_empty_dir()
-    pkg_scripts_fpath = os.path.join(os.path.dirname(
+    pkg_scripts_search_fpath = os.path.join(os.path.dirname(
```

Comment:
This is a search string for glob and not a file path. The `fpath` at the end could lead to confusion.

```suggestion
    pkg_scripts_search_str = os.path.join(os.path.dirname(
```